RadixWallet/Clients/SecurityCenterClient/SecurityCenterClient+Interface.swift

```diff
@@ -0,0 +1,49 @@
+import CloudKit
+import ComposableArchitecture
+import DependenciesAdditions
+import os
+
+// MARK: - SecurityCenterClient
+public struct SecurityCenterClient: DependencyKey, Sendable {
+	public let problems: Problems
+	public let lastManualBackup: LastManualBackup
+	public let lastCloudBackup: LastCloudBackup
+}
+
+// MARK: SecurityCenterClient.Problems
+extension SecurityCenterClient {
+	public typealias Problems = @Sendable () async -> AnyAsyncSequence<[SecurityProblem]>
+	public typealias LastManualBackup = @Sendable () async -> AnyAsyncSequence<BackupStatus>
+	public typealias LastCloudBackup = @Sendable () async -> AnyAsyncSequence<BackupStatus>
+}
+
+// MARK: - SecurityProblem
+public enum SecurityProblem: Hashable, Sendable, Identifiable {
+	case problem3(accounts: Int, personas: Int)
+	case problem5
+	case problem6
+	case problem7
+	case problem9
```

Comment:
I would add as a comment of each the description on [Confluence](https://radixdlt.atlassian.net/wiki/spaces/AT/pages/3392569357/Security-related+Problem+States+in+the+Wallet)